docs/how-to/join-a-network.md

```diff
@@ -119,8 +120,9 @@ Deadline Challenge:   434 (33m0s ago)
 Deadline FaultCutoff: 384 (58m0s ago)
 ```
 
-In this next example, the miner can be safely restarted because no Deadlines are earlier than Current Epoch of 497. You have ~45 minutes before the miner must be back online to declare faults (FaultCutoff). If the miner has no faults, you have about an hour.
-```
+In this next example, this miner should **not** be shut down or restarted since there ar 768 sectors being challenged right now. 
```

Comment:
```suggestion
In this next example, this miner should **not** be shut down or restarted since there are 768 sectors being challenged right now. 
```